verticapy/tests_new/conftest.py

```diff
@@ -14,16 +14,245 @@
+    pred = list(
+        np.random.choice([0, 1], size=first_count, p=[1 - prob_1_first, prob_1_first])
+    )
+    pred.extend(
+        np.random.choice(
+            [0, 1], size=second_count, p=[1 - prob_1_second, prob_1_second]
+        )
+    )
+    pred.extend(
+        np.random.choice([0, 1], size=third_count, p=[1 - prob_1_third, prob_1_third])
+    )
+    prob = np.linspace(0, 1, count)
+    dummy = verticapy.vDataFrame({"y_true": prob, "y_score": pred})
+    yield dummy
+
+
+@pytest.fixture(scope="session")
+def dummy_dist_vd():
+    """
+    Create a dummy vDataFrame
+    """
+    data_size = DUMMY_TEST_SIZE
+    ones_percentage = 0.4
+    median = 50
+    qrtrs_1_3 = [40, 60]
+    percentages = [0.4, 0.3, 0.3]
+    categories = ["A", "B", "C"]
+    category_array = np.random.choice(
+        categories, size=data_size, p=[percentages[0], percentages[1], percentages[2]]
+    )
+    category_array = category_array.reshape(len(category_array), 1)
+    result_array = np.concatenate(
+        (
+            np.zeros(int(data_size * (1 - ones_percentage))),
+            np.ones(int(data_size * ones_percentage)),
+        )
+    )
+    np.random.shuffle(result_array)
+    result_array = result_array.reshape(len(result_array), 1)
+    std = (qrtrs_1_3[1] - qrtrs_1_3[0]) / (2 * np.sqrt(2) * erfinv(0.5))
+    data = np.random.normal(median, std, data_size)
+    data_2 = np.random.normal(median + 10, std + 5, data_size)
+    data = data.reshape(len(data), 1)
+    data_2 = data_2.reshape(len(data_2), 1)
+    data[-1] = data.max() + 15
+    data_2[-1] = data_2.max() + 15 + 10
+    data_all = pd.DataFrame(
+        np.concatenate((data, data_2, result_array, category_array), axis=1),
+        columns=["0", "1", "binary", "cats"],
+    )
+    with tempfile.TemporaryDirectory() as temp_dir:
+        dummy = verticapy.read_pandas(data_all, temp_path=temp_dir)
+    dummy["binary"].astype("int")
+    yield dummy
+
+
+@pytest.fixture(scope="session")
+def dummy_line_data_vd():
+    """
+    Create a dummy vDataFrame that has time data
+    """
+    data_size = int(DUMMY_TEST_SIZE / 10)
+    start_year = 1900
+    step = 10
+    num_values = data_size
+    years = [start_year + i * step for i in range(num_values)] * 2
+    values = [random.randint(1, 100) for _ in range(data_size * 2)]
+    category = ["A"] * data_size + ["B"] * data_size
+    yield verticapy.vDataFrame({"date": years, "values": values, "category": category})
+
+
+@pytest.fixture(scope="session")
+def dummy_pred_data_vd():
+    """
+    Create a dummy vDataFrame that has prediction as a column
+    """
+    data_size = DUMMY_TEST_SIZE
+    x_val = np.linspace(0, 10, data_size)
+    y_val = np.linspace(0, 10, data_size)
+    z_val = np.linspace(0, 10, data_size)
+    pred = [0 if x_val[i] + y_val[i] + z_val[i] > 20 else 1 for i in range(len(x_val))]
+    yield verticapy.vDataFrame({"X": x_val, "Y": y_val, "Z": z_val, "Category": pred})
+
+
+@pytest.fixture(scope="module")
```

Comment:
Why the scope of the fixtures from this point becomes "module"? They don't seem to be called anywhere in this module!

Reply:
1) Why the scope is module? 
So that they are reloaded for each module. Just in case some of the tests are performing any operations on the data itself. The previous datasets were made for specific tests and because I know I am not performing any operations on those tests, i chose the scope to be "session" to increase efficiency. 

2) Not used?
They are a list of datasets that can be used for testing. All of them will be used in different tests. Current PR is limited to only plotting library tests for highcharts.